os2forms_rest_api.info.yml

```diff
@@ -0,0 +1,9 @@
+name: 'OS2Form REST API'
+type: module
+description: 'OS2Form REST API'
+package: Web services
```

Comment:
matter of taste, but shouldn't it be OS2Forms for package?

Reply:
Fixed in https://github.com/OS2Forms/os2forms_rest_api/pull/1/commits/e217fd6dc093cb028ab7ca39bb1b41a669aefd01.